tests/jerry/es2015/typedarray-prototype-indexof.js

```diff
@@ -0,0 +1,64 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
Please add a test where this new `indexof` method is invoked on a non typedarray object.

Reply:
Please add tests where the `fromIndex` has a negative value.